pandas_gbq/gbq.py

```diff
@@ -483,13 +483,16 @@ def run_query(self, query, max_results=None, progress_bar_type=None, **kwargs):
                 project=self.project_id,
             )
             logger.debug("Query running...")
-        except (RefreshError, ValueError):
+        except (RefreshError, ValueError) as ex:
             if self.private_key:
-                raise AccessDenied("The service account credentials are not valid")
+                raise AccessDenied(
+                    "The service account credentials are not " f"valid: {ex}"
```

Comment:
This string is short enough, is it worth avoid the concat and just using the whole thing as a format string?

Reply:
Good catch! I forgot to double check after the black reformat.